schemas/activity/tissueCulturePreparation.schema.tpl.json

```diff
@@ -13,8 +13,10 @@
             ]
         },
         "cultureMedium": {
-            "type": "string",
-            "_instruction": "Solution the culture was maintained in prior to the experiment"
+            "_instruction": "Solution the culture was maintained in prior to the experiment",
```

Comment:
I think this needs another verb, e.g. "Specify the solution...", but for consistency I suggest we check over all the instructions in ephys together after this is merged, and update them in a later PR